keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md

```diff
@@ -537,12 +512,14 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
-| API server | KCM | Behavior                                                                                                                   |
-|------------|-----|----------------------------------------------------------------------------------------------------------------------------|
-| off | off | Existing Kubernetes behavior.                                                                                              |
-| on | off| Existing Kubernetes behavior, only users can set `pvc.Status.LastPhaseTransitionTime`.                        |
-| off | on | PV controller may try to change `pvc.Status.LastPhaseTransitionTime`, which will fail on the API server. |
-| on | on | New behavior.                                                                                                              
+Version skew is not applicable, KCM was not changed in scope of this enhancement.
+
+| API server | KCM | Behavior              |
```

Comment:
let's remove the KCM column from the table - it doesn't matter

Reply:
Removed.